### ISSUE TITLE
review

### ISSUE THREAD

**Issue by ak0327** [ak0327/42_multilayer-perceptron#6]
* notebookにまとめるなど
* 説明要件 You will be asked to explain to your corrector the notions of 
  - [ ] feedforward
  - [ ] backpropagation
  - [ ] gradient descent
